components/authz-service/server/v2/project.go

```diff
@@ -123,7 +123,7 @@ func (s *ProjectState) CreateProject(ctx context.Context,
 		return nil, status.Errorf(codes.InvalidArgument,
 			"creating project with ID %q: %s", req.Id, err.Error())
 	}
-	resp, err := s.store.CreateProject(ctx, &p)
+	resp, err := s.store.CreateProject(ctx, &p, true)
```

Comment:
And this is the main thoroughfare for users, so it should always create policies (at least for this iteration).